### DISCUSSION TITLE
stop button

### DISCUSSION THREAD

**Discussion by apricotandpearjam** [jamland/samplescope#146] [Ideas]
first of all.
THANKYOU for making such an incredibly useful tool for us.
I am very grateful for this.

May I suggest a stop button or at least for the spacebar to be a start/stop toggle and also I find you have to click the play buttons twice for the audio to begin.

cheers!!

**Comment by jamland** [comment]
Hi @apricotandpearjam !

Thank you for the feedback.
I'm still trying to figure out the best way to control the player so it will stay visually clean and simple but well accessible same time.

At the moment basic navigation implemented with arrows, you can use it:

➡️  ⬅️   - PLAY / STOP  focused sample
⬆️  ⬇️  - move in the list up / down

Probably, I also need to think about how to showcase this within the app.

**Comment by apricotandpearjam** [comment]
Thanks for your reply. 
I would recommend space bar to be the start / stop. 
Is that possible?

**Reply by jamland** [reply]
Yes, sure it is possible. Will try it

**Reply by joachime** [reply]
I second using the space bar for start and stop playback.

**Comment by jamland** [comment]
✅   Added this functionality in the scope of this update: https://github.com/jamland/samplescope/discussions/206
